PrxDecrypter.cpp

```diff
@@ -1020,33 +1047,33 @@ int pspDecryptPRX(const u8 *inbuf, u8 *outbuf, u32 size, const u8 *seed)
 	// since we don't know the PRX type we attempt a decrypt using all
 	auto res = pspDecryptType0(inbuf, outbuf, size);
 	if (res >= 0) {
-		printf("Decryption successful for tag %02X with type 0\n", tag);
+		//printf("Decryption successful for tag %02X with type 0\n", tag);
```

Comment:
Context: was attempting to map which PRXs use which key - do you know a better way to identify which key to use?

Reply:
No, I just removed the line because it became very spammy when extracting PSAR's — I'll add a verbose flag for it (which could be given as an argument to the program in the future) :)